tests/pool/test_pool_reader.py

```
@@ -1,53 +1,52 @@
+    reader.close()
+    reader.close()
+
+
+def test_context_manager_calls_close() -> None:
+    runtime = MagicMock()
+    with PoolReader.from_runtime(runtime, schema=_SCHEMA) as reader:
+        assert reader.pool_name == _SCHEMA.name
+
+
+@patch("dr_llm.pool.reader.load_schema", return_value=None)
+def test_open_raises_when_pool_not_in_catalog(mock_load: MagicMock) -> None:
```

Ruff (0.15.12)

[warning] Unused function argument: `mock_load`

(ARG001)